manager2/src/app/callback/registered/registered.component.html

```diff
@@ -5,9 +5,11 @@ <h2 class="panel-title">Congratulation</h2>
         </div>
         <div class="panel-body">
             <p>Your account creation has been taken into account.</p>
+            <br>
             <p>You will receive an email soon to <b>confirm your email address</b>.</p>
+            <p style='color: red'><b> Please check your email and your spam folder ! </b></p
```

Comment:
Could use bootstrap label warning for styling

Reply:
yes